site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,395 @@
+				id="sc-three-1"
+				role="tab"
+				aria-selected="true"
+				data-segmented-control
+			>
+				Define Strategy
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-2"
+				role="tab"
+				data-segmented-control
+			>
+				Plan
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-3"
+				role="tab"
+				data-segmented-control
+			>
+				Prepare
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-4"
+				role="tab"
+				data-segmented-control
+			>
+				Adopt
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-5"
+				role="tab"
+				data-segmented-control
+			>
+				Govern
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-6"
+				role="tab"
+				data-segmented-control
+			>
+				Manage
+			</button>
+		</div>
+	</div>
+
+	<div role="tabpanel" aria-labelledby="sc-three-1">
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 1</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-2" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 2</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-3" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 3</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-4" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 4</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-5" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 5</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-6" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+
+	<!-- After Tabpanels -->
+	<div class="segmented-controls-footer border-top buttons padding-top-sm">
+		<button class="button button-sm button-primary" data-segmented-control-nav="previous">
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						class="fill-current-color"
+						d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"
+					/>
+				</svg>
+			</span>
+			<span>Previous</span>
+		</button>
+
+		<button class="button button-sm button-primary button-filled" data-segmented-control-nav="next">
+			<span>Next</span>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="-255 57 448 448">
+					<path
+						class="fill-current-color"
+						d="M-253.625 281v-32H131.75L-16.938 100.313 5.687 77.688 193 265 5.687 452.313l-22.625-22.625L131.75 281h-385.375z"
+					/>
+				</svg>
+			</span>
+		</button>
+	</div>
+</tab-container>
+```
+
+## Size modification
+
+Using the `segmented-controls-lg` can be used to make segmented control bigger than the default.
```

Comment:
This size seems like an issue on mobile. It's so big that the center button is usually cut off. Do we need it? Could we instead just ensure it scales the way we want from narrow to widescreens? (The default version seems to do this quite well).

Reply:
I agree.

I think forcing it down in size to the standard, at mobile, would be a good compromise.  That way the larger views can still use the bigger version.

Reply:
I went ahead and removed, based on the comps I dont see the reason for the large variant.  Rather just the nice responsive nature we already have.